apps/src/aichat/redux/utils.ts

```diff
@@ -75,6 +75,22 @@ export const hasFilledOutModelCard = (modelCardInfo: ModelCardInfo) => {
   return true;
 };
 
+export const emptyModelCard = (modelCardInfo: ModelCardInfo) => {
+  let allFieldsEmpty = true;
+  for (const key of getTypedKeys(modelCardInfo)) {
+    if (key === 'isPublished') {
+      continue;
+    } else if (key === 'exampleTopics') {
+      if (modelCardInfo['exampleTopics'].length !== 0) {
+        allFieldsEmpty = false;
+      }
+    } else if (modelCardInfo[key] !== '') {
+      allFieldsEmpty = false;
```

Comment:
nit: early `return false` here rather than continuing to loop through all fields?

Also actually I think we have an `EMPTY_MODEL_CARD` constant so we could just deep compare / stringify and compare to that?